src/controllers/languages.js

```
@@ -0,0 +1,93 @@
+const Language = require('~/models/languages')
```

ESLint

[error] 'Language' is assigned a value but never used.

(no-unused-vars)